src/inline-help.scss

```diff
@@ -2,29 +2,18 @@
 @import "./functions";
```

Comment:
if the component is not relying on the old variables, then it has to import `./new-settings` instead `./settings`

Reply:
Done